yarn.lock

```diff
@@ -2548,6 +2548,13 @@
     npmlog "^6.0.2"
     write-file-atomic "^4.0.1"
 
+"@littlespoon/icons@1.8.0":
```

Comment:
This should not be added, you will need to update to latest version in your package's deps

Reply:
Fixed in cf62f6651f04b78f882663b6401a8ca4df30fe29